pkg/chart/chart_test.go

```diff
@@ -78,3 +78,21 @@ func TestSaveChartNoRawData(t *testing.T) {
 
 	is.Equal([]*File(nil), res.Raw)
 }
+
+func TestMetadata(t *testing.T) {
+	chrt := Chart{
+		Metadata: &Metadata{
+			Name:       "foo.yaml",
+			AppVersion: "1.0.0",
+			APIVersion: "2.0.0",
```

Comment:
Do you mind setting it as follows: `APIVersion: "v2"`? It is to signify the chart v2 format